README.md

```diff
@@ -119,6 +119,14 @@ Please note that currently the SQL backend does not support submitted responses
 
 ---
 
+To use `EQ_STORAGE_BACKEND` as `datastore` or `EQ_SUBMISSION_BACKEND` as `gcp` directly on GCP and not a docker image, you need to set the GCP project using the following command:
```

Comment:
`EQ_SUBMISSION_BACKEND` is `gcs`, rather than `gcp`

Reply:
Eagle eyes, changed. 👍